server/queuer.js

```diff
@@ -13,7 +17,7 @@ function sleep(ms) {
 }
 
 function printPush(push) {
-	return `${push.sha} at ${push.created_at} by ${push.author}: ${push.message}`;
+	return push ? `${push.sha} at ${push.created_at} by ${push.author}: ${push.message}` : '—';
```

Comment:
This fix is pretty much just hiding some other bug — maybe when the DB was completely empty I got undefined `push` somehow?

Reply:
That's very likely -- I vaguely remember ignoring special cases where there are no existing pushes. Now, after a few months, that neglect comes back to haunt me 😨